### DISCUSSION TITLE
Transmission container stops immediately after starting it

### DISCUSSION THREAD

**Discussion by blQSheep** [haugene/docker-transmission-openvpn#2841] [Q&A] [answered]
Hi i can't get transmission up & running because as soon as i start the container it stops after 2 seconds.
I already went trough the [FAQs](https://haugene.github.io/docker-transmission-openvpn/faq/) and coudn't find anything related. Please help. 

CONTAINER ID   IMAGE                                        COMMAND                  CREATED         STATUS
        PORTS
                                 NAMES
02cacb6257b6   haugene/transmission-openvpn                 "dumb-init /etc/open…"   8 seconds ago   Exited (1) 6 seconds ago
                                 transmission-openvpn-transmission-openvpn-1

version: '3.3'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - /media/NAS/sharing/Downloads:/data
            - ./config:/config
        environment:
            - TZ=Europe/Berlin
            - OPENVPN_PROVIDER=IPVANISH
            - OPENVPN_CONFIG=denmark,germany,netherlands,poland
            - OPENVPN_USERNAME=REMOVED
            - OPENVPN_PASSWORD=REMOVED
            - LOCAL_NETWORK=192.168.178.0/24
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - 9091:9091
        image: haugene/transmission-openvpn

**Comment by pkishino** [comment]
You need to provide at least some logs for any help please

**Reply by blQSheep** [reply]
Hi thanks for your response, here is the log

 docker logs 02cacb6257b6
Starting container with revision: 07f5a2b9aea5028c9bb75438c1552708e91dde71
TRANSMISSION_HOME is currently set to: /config/transmission-home
Creating TUN device /dev/net/tun
Using OpenVPN provider: IPVANISH
Running with VPN_CONFIG_SOURCE auto
Provider IPVANISH has a bundled setup script. Defaulting to internal config
Executing setup script for IPVANISH
Downloading OpenVPN config bundle configs.zip into temporary file /tmp/tmp.DjusHTL5bZ
Extract OpenVPN config bundle into /etc/openvpn/ipvanish
4 servers found in OPENVPN_CONFIG, poland chosen randomly
Supplied config poland.ovpn could not be found.
Your options for this provider are:
default.ovpn
ipvanish-AE-Dubai-dxb-c01.ovpn
ipvanish-AE-Dubai-dxb-c02.ovpn
ipvanish-AE-Dubai-dxb-c03.ovpn
ipvanish-AE-Dubai-dxb-c04.ovpn
ipvanish-AE-Dubai-dxb-c05.ovpn
ipvanish-AE-Dubai-dxb-c06.ovpn
ipvanish-AE-Dubai-dxb-c07.ovpn
ipvanish-AE-Dubai-dxb-c08.ovpn
[list goes on until end]

**Reply by pkishino** [reply]
OPENVPN_CONFIG=denmark,germany,netherlands,poland
isn’t going to work.. you’ll have to check the config names..

**Reply by blQSheep** [reply]
@pkishino thanks, i've sensed that it would probably be that setting.
I got that from the [documentation](https://haugene.github.io/docker-transmission-openvpn/run-container/):

"Pro tip: choose multiple servers. For example: OPENVPN_CONFIG=france,sweden,austria,italy,belgium This will ensure a location near you, but at the same time, it will allow some redundancy. Set Docker to restart the container automatically and you have a failover mechanism. The container chooses one of the configs at random when it starts and it will bounce from server to server until it finds one that works."

So how is it possible for me to configure a variable in OPENVPN_CONFIG, so that the openvpn chooses among a set of countries randomly or the best available?

**Reply by pkishino** [reply, marked as answer]
See the list of names or prints?
Like ipvanish-AE-Dubai-dxb-c01.ovpn
You can choose several of those and use ipvanish-AE-Dubai-dxb-c01 as the name

**Comment by blQSheep** [comment]
OK, understood,

How do i know which is the best of the specific country, city server ipvanish-AE-Dubai-dxb-cXX ??

Like is the highest server # the best server for a specific city?

 

Von: Patrick Kishino ***@***.***> 
Gesendet: Dienstag, 21. Mai 2024 02:52
An: haugene/docker-transmission-openvpn ***@***.***>
Cc: blQSheep ***@***.***>; Author ***@***.***>
Betreff: Re: [haugene/docker-transmission-openvpn] Transmission container stops immediately after starting it (Discussion #2841)

 

See the list of names or prints?
Like ipvanish-AE-Dubai-dxb-c01.ovpn
You can choose several of those and use ipvanish-AE-Dubai-dxb-c01 as the name

**Comment by pkishino** [comment]
That you will have to check with the provider itself/ BRPatrick KishinoOn 21 May 2024, at 01:58, blQSheep ***@***.***> wrote:﻿
OK, understood,



How do i know which is the best of the specific country, city server ipvanish-AE-Dubai-dxb-cXX ??



Like is the highest server # the best server for a specific city?



 



Von: Patrick Kishino ***@***.***> 

Gesendet: Dienstag, 21. Mai 2024 02:52

An: haugene/docker-transmission-openvpn ***@***.***>

Cc: blQSheep ***@***.***>; Author ***@***.***>

Betreff: Re: [haugene/docker-transmission-openvpn] Transmission container stops immediately after starting it (Discussion #2841)



 



See the list of names or prints?

Like ipvanish-AE-Dubai-dxb-c01.ovpn

You can choose several of those and use ipvanish-AE-Dubai-dxb-c01 as the name
